lightweight-term-count-update.php

```diff
@@ -278,6 +283,26 @@ public function transition_type( $new, $old ) {
 			return false;
 		}
 	}
+
+	/**
+	 * Force-recount posts for a term.  Do this only when the update originates from the edit term screen.
+	 *
+	 * @param  int    $term_id the term id.
+	 * @param  int    $tt_id the term taxonomy id.
+	 * @param  string $taxonomy the taxonomy.
+	 *
+	 * @return bool false if the screen check fails, true otherwise
+	 */
+	public function maybe_recount_posts_for_term( $term_id, $tt_id, $taxonomy ) {
+		$screen = function_exists( 'get_current_screen' ) ? get_current_screen() : '';
+		if ( 'WP_Screen' !== get_class( $screen ) ) {
```

Comment:
[`isntanceof`](http://php.net/manual/en/internals2.opcodes.instanceof.php) is quite a bit faster than running `get_class()` and doing a string comparison against the result. Is there any reason I'm not seeing to use that instead?